File: network-client/src/main/java/com/mgtriffid/academ/network/client/CommandsReceived.java
package com.mgtriffid.academ.network.client;

import com.mgtriffid.academ.network.common.commands.ConnectionCommand;

import java.util.List;

/**
 * Created by mgtriffid on 31.03.17.
 */
public class CommandsReceived {
    private final List<ConnectionCommand> connectionCommands;

    public CommandsReceived(List<ConnectionCommand> connectionCommands) {
        this.connectionCommands = connectionCommands;
    }

    public List<ConnectionCommand> getConnectionCommands() {
        return connectionCommands;
    }
}

File: network-client/src/main/kotlin/com/mgtriffid/academ/network/client/CommandsReceived.kt
package com.mgtriffid.academ.network.client

import com.mgtriffid.academ.network.common.commands.ConnectionCommand
import com.mgtriffid.academ.network.common.commands.GameLogicCommands

/**
 * Created by mgtriffid on 31.03.17.
 */
class CommandsReceived(
        val connectionCommands: List<ConnectionCommand>,
        val gameLogicCommands: List<Pair<Int, GameLogicCommands>>
)
